src/models/Settings.php

```diff
@@ -32,6 +32,11 @@ class Settings extends Model
     public $allowAttachments = false;
 
     /**
+     * @var bool
+     */
+     public $includeAllData = false;
```

Comment:
Not sure a setting is necessary for this. I'm sure most people would be fine to have the additional info in the body.